src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -41,15 +41,20 @@
         'list.language': 'Language'|trans({}, 'Admin.Global'),
         'list.example': 'Example'|trans({}, 'Admin.Global'),
         'list.edit': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
-        'list.reset': 'Reset settings'|trans({}, 'Admin.Actions'),
-        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature')
+        'list.reset': 'Reset'|trans({}, 'Admin.Actions'),
+        'list.reset.success': 'Your symbol and format customizations have been successfully reset for this language.'|trans({}, 'Admin.Notifications.Success'),
+        'list.example.format': '%price%'|trans({}, 'Admin.International.Feature'),
+        'modal.restore.title': 'Are you sure you want to restore default settings?'|trans({}, 'Admin.International.Feature'),
+        'modal.restore.apply': 'Restore'|trans({}, 'Admin.Actions'),
+        'modal.restore.cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+        'modal.restore.body': 'Restoring your currency\'s default settings will delete all the customizations you made before. Parameters will look just the same as when the currency is freshly imported.'|trans({}, 'Admin.International.Feature')
       }|json_encode|replace("'", "&#39;")|raw
     }
   }) }}
 
   <div class="card">
     <div class="card-header">
-      {{ 'Currency'|trans({}, 'Admin.Global') }}
+      {{ 'Currencies'|trans({}, 'Admin.Global') }}
```

Comment:
Ok for keeping a generic name but why plural since the form is dedicated to only one currency?

Reply:
It is pure consistency; it is the case for all the localization pages of PrestaShop, when you click the 'Add new something' button in the top right-hand corner, see examples below.

<img width="634" alt="Capture d’écran 2020-02-19 à 18 18 53" src="https://user-images.githubusercontent.com/32565890/74857722-64aa8600-5344-11ea-99e6-f1443c4cf693.png">

<img width="546" alt="Capture d’écran 2020-02-19 à 18 19 09" src="https://user-images.githubusercontent.com/32565890/74857732-683e0d00-5344-11ea-9ac0-52a7e20652ec.png">

<img width="644" alt="Capture d’écran 2020-02-19 à 18 19 19" src="https://user-images.githubusercontent.com/32565890/74857743-6b38fd80-5344-11ea-8bd6-4c295004783e.png">